Pipfile

```diff
@@ -0,0 +1,23 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+certifi = "==2022.6.15"
+cffi = "==1.15.0"
+charset-normalizer = "==2.0.12"
+deprecated = "==1.2.13"
+idna = "==3.3"
+pycparser = "==2.21"
+pygithub = "==1.55"
+pyjwt = "==2.4.0"
+pynacl = "==1.5.0"
+requests = "==2.28.0"
+urllib3 = "==1.26.9"
+wrapt = "==1.14.1"
+
+[dev-packages]
+
+[requires]
+python_version = "3.7"
```

Comment:
I chose version 3.7 here because that is the oldest version that is still supported.